### PR TITLE
[APG-1084] Add array_superstructure_srvs to bridged messages

### PR DESCRIPTION
**Pull request by bmagyar**
Builds on top of #20

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -34,23 +34,24 @@
   <build_depend>xmlrpcpp</build_depend>
 
   <!-- Bridged Message Types -->
+  <build_depend>actionlib_msgs</build_depend>
+  <build_depend>array_superstructure_srvs</build_depend>
```
Comment (carlos-m159) [inline package.xml:38]: This might require listing `array_superstructure_msgs` here as well?

Comment (svwilliams) [reply package.xml:38]: I was just about to ask if there was a ROS1 version of the `array_superstructure_srvs` included in the build. I'm guessing "no".

I know we have a fancy "automatically convert ROS1 messages to ROS2" package here: https://github.com/locusrobotics/locus_ros2_messages

I'm not sure if going the opposite direction is supported.

Comment (carlos-m159) [reply package.xml:38]: Ah, right... I assumed it was a ROS1 pkg. It is going to be a problem then

Comment (bmagyar) [reply package.xml:38]: The ROS1 versions also exist, located here: https://github.com/locusrobotics/locus_vector/tree/devel/vector_common

File: package.xml [hunk 2]
```diff
@@ -35,6 +35,7 @@
 
   <!-- Bridged Message Types -->
   <build_depend>actionlib_msgs</build_depend>
+  <build_depend>array_superstructure_srvs</build_depend>
```
Comment (svwilliams) [inline package.xml:38]: The ROS1 Bridge message dependency system is not recursive. You will need to manually add all dependencies of `array_superstructure_srvs` to this list as well. At a minimum, you need `array_superstructure_msgs` in this list. But that is as far as I looked. There may be others.

**Review by svwilliams — CHANGES_REQUESTED** [review]

**Review by carlos-m159 — APPROVED** [review]

File: package.xml (hunk 2 above)
Comment (bmagyar) [reply package.xml:39]: Done!

**Review by svwilliams — APPROVED** [review]
Build server looks happy. Ship it.

**Comment by bmagyar** [comment]
Thanks for the tips guys!

**Pull request merged**